src/test/java/LoginTests.java

```diff
@@ -1,25 +1,23 @@
+import Pages.HomePage;
+import Pages.LoginPage;
+import Pages.BasePage;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import org.openqa.selenium.*;
 
 import java.time.Duration;
 
 public class LoginTests extends BaseTest {
     @Test
-    public void LoginEmptyEmailPasswordTest() {
+    public void LoginValidCredentialsTest() {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example  public void **loginValidCredentialsTest()**